### PR TITLE
build: add clickhouse integration testing

### PR DESCRIPTION
**Pull request by Ian2012**

### CODE REVIEW

File: tutor-contrib-harmony-plugin/tutor_k8s_harmony_plugin/patches/kustomization [hunk 1]
```diff
@@ -19,6 +19,7 @@ patches:
 - path: plugins/k8s_harmony/k8s/deployment-revision-history.yaml
   target:
     kind: Deployment
+{%- if K8S_HARMONY_ENABLE_SHARED_SEARCH_CLUSTER %}
```
Comment (bmtcril) [inline tutor-contrib-harmony-plugin/tutor_k8s_harmony_plugin/patches/kustomization:22]: Is this related?

Comment (Ian2012) [reply tutor-contrib-harmony-plugin/tutor_k8s_harmony_plugin/patches/kustomization:22]: It just fixes an issue when ES is disabled. Another PR was open with the fix. Will rebase once is merged
